channel.json

```diff
@@ -120,6 +120,7 @@
 		"https://raw.githubusercontent.com/x3ns/x3-alpha-theme/master/packages.json",
 		"https://raw.githubusercontent.com/xgenvn/InputHelper/master/packages.json",
 		"https://raw.githubusercontent.com/zfkun/sublime-kissy-snippets/master/packages.json",
+		"https://git.loverde.com.br/git/LC-Dark-Color-Scheme/src/main/package.json",
```

Comment:
If you're adding to the channel, you don't need to also add to the repository (IIRC, admittedly we don't get this often).

I'm getting a "this connection is note private" warning on that URL though, not sure that will work.

Reply:
Right. The format is not correct though. Have a look at this for example: https://raw.githubusercontent.com/zfkun/sublime-kissy-snippets/master/packages.json

Since you're using something other than the public services we can crawl, you need to provide a pre-packages sublime package (zip), and a json describing the package according this document:
https://raw.githubusercontent.com/wbond/package_control/master/example-repository.json